### ISSUE TITLE
Reviewing UI/UX in the Add-ons page for Free Flow

### ISSUE THREAD

**Issue by pablohoneyhoney** [Automattic/wp-calypso#65416]
This is the flow I'm currently seeing (text related notes in #65410)

<img width="1224" alt="Screen Shot 2022-07-08 at 9 38 18 AM" src="https://user-images.githubusercontent.com/25516611/178058425-c1d0a8cd-0980-4522-9a9a-22beb9f8aa52.png">
<img width="1296" alt="Screen Shot 2022-07-08 at 9 38 24 AM" src="https://user-images.githubusercontent.com/25516611/178058433-743c5dcb-4c95-4ff5-b809-235a28c40363.png">
<img width="1285" alt="Screen Shot 2022-07-08 at 9 37 32 AM" src="https://user-images.githubusercontent.com/25516611/178058443-3c04bb24-122e-4e7a-9980-051de02b2ccb.png">

- [ ] `Continue` should be `Skip` since Continue has an implicit action first you don't need to make in reality. If we rather want people to take an action, we could either hide temporarily Continue
- [ ] Select all add-ons should be more discreet. Without + and only Select all.
- [ ] `Add to my plan` > `Select/Add/Get/Buy (name of the add-on) `
- [ ] The icons aren't fully representative of the add-on. Like the CSS or the Ads-Free. Premium Themes should be connected to how we highlight them in the design picker/showcase or more related to a layout icon. We don't use that brush elsewhere.
- [ ] Once you select one or multiple, we should surface the price alongside the check out button to give anticipation of what you are paying for, which you don't see until you we load the check out page. 
- [ ] `Check out` > `Continue to pay` perhaps.
- [ ] It'd be better to have a layout of horizontal rows so it's more scalable to any number of add-ons. At least of we add more than two in. 

@lucasmendes-design @simison @ianstewart @michaelpick

**Comment by DavidRothstein** [comment]
There's also a UX issue here on mobile specifically -- once you scroll down to the bottom of the list and select an add-on, you are kind of stuck, until you realize you have to scroll all the way up again to find the new "Go to checkout" button.

Probably some kind of checkout option towards the bottom of the screen would help with this (and might be somewhat helpful at desktop widths too).

This is what you see currently after selecting an add-on at the bottom of the list:

![step3](https://user-images.githubusercontent.com/235183/178068227-a79fbc2d-c4c8-4c3f-9366-37b4ac5e159b.png)

**Comment by lucasmendes-design** [comment]
> The icons aren't fully representative of the add-on. Like the CSS or the Ads-Free. Premium Themes should be connected to how we highlight them in the design picker/showcase or more related to a layout icon. We don't use that brush elsewhere.

Agreed, The CSS I tried to use something like "< >" because it's similar to HTML and CSS. I'll find a better one, otherwise, I'll do a specific icon for each one. 

> Once you select one or multiple, we should surface the price alongside the check out button to give anticipation of what you are paying for, which you don't see until you we load the check out page.

Got it! 

> It'd be better to have a layout of horizontal rows so it's more scalable to any number of add-ons. At least of we add more than two in.

I think we need to balance the layout and how we can sell the add-ons with enough space, I'll iterate on that for the next launch when we have more add-ons to offer. 

> There's also a UX issue here on mobile specifically -- once you scroll down to the bottom of the list and select an add-on, you are kind of stuck, until you realize you have to scroll all the way up again to find the new "Go to checkout" button.

Thanks for this @DavidRothstein. It makes a lot of sense to have a better interaction for mobile. I'll work on that 🙌🏼

**Comment by lucasmendes-design** [comment]
> Add to my plan > Select/Add/Get/Buy (name of the add-on)

I followed the same pattern on the pricing page (Choose + Name of the add-on). 

> Select all add-ons should be more discreet. Without + and only Select all.

✔

> The icons aren't fully representative of the add-on. 

I'm creating new icons for Free Ads

> Once you select one or multiple, we should surface the price alongside the check out button to give anticipation ....

✔

### **Web flow**

Once the user select a add-on, the button changes to secondary to primary. If we want to create friction we can also try to use the Skip action on subtitle, the same rationale for Pricing page ([example](https://d.pr/i/165B6B))

https://user-images.githubusercontent.com/98343583/178377151-0256a33f-ec3c-498e-a201-5816d0a47f92.mov

### **Mobile flow (@DavidRothstein)**

For mobile, we can use a different approach to show the price alongside the purchase button. 

https://user-images.githubusercontent.com/98343583/178378741-08ba3b72-3270-46cc-8580-61a7ffc72bc0.mov

**Comment by lucasmendes-design** [comment]
Btw, I think we should get rid of that or use the same links as [WordPress Support Add-on](https://wordpress.com/support/add-ons/) for the help center, otherwise, it's out of context..  cc @DavidRothstein @gmovr 

<img width="654" alt="image" src="https://user-images.githubusercontent.com/98343583/178482611-1119acca-616b-4c02-820f-a37c9730e835.png">

<img width="416" alt="image" src="https://user-images.githubusercontent.com/98343583/178481749-844be005-4f40-4391-9bf5-f8d631ea2b5f.png">

**Comment by lucasmendes-design** [comment]
**Collection of icons:** 
<img width="419" alt="image" src="https://user-images.githubusercontent.com/98343583/179036614-69a956ad-d806-4277-bcae-5368eb2f2b6e.png">

**Icons in context:** 
<img width="877" alt="image" src="https://user-images.githubusercontent.com/98343583/179036900-4890b8eb-9a38-4463-bc15-1d33e04e85e0.png">

**Comment by lucasmendes-design** [comment]
Do we have an update for that? cc: @gmovr
